### ISSUE TITLE
[Feature] Is it possible to turn off Farming Tooltips only?

### ISSUE THREAD

**Issue by pinetreemoonlight** [Annosz/UIInfoSuite2#350]
I find the luck and other info displayed essential, but the farming tooltips are distracting to me. Is there a way to disable them?

**Comment by drewhoener** [comment]
@pinetreemoonlight it should be possible, do you mean the ones that say how long a crop has left to grow?

**Comment by pinetreemoonlight** [comment]
> @pinetreemoonlight it should be possible, do you mean the ones that say how long a crop has left to grow?

Yes exactly. While the information is useful, I somehow prefer to just watch the crops grow and be surprised one day when they are ready to farm if that makes sense haha. So it would be great if it was an option to turn the crop tooltips off. Thank you so much for considering it.
